app/controllers/api_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApiController < ApplicationController
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :null_session
```

Comment:
As a best practice, I prefer `protect_from_forgery unless: -> { request.format.json? }`. But this works too.